### DISCUSSION TITLE
Welcome to learngo Discussions!

### DISCUSSION THREAD

**Discussion by inancgumus** [inancgumus/learngo#44] [General]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Show your Go programs.
  * Ask for feedback.
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JaiminBrahmbhatt** [comment]
You can add the next --> and <-- the previous lesson buttons at the end of each lesson, making it very fast for any learner to go up and down in a lesson from the markdown file itself. If you want I am free to help. Loved the content <3

**Reply by inancgumus** [reply]
Hey @JaiminBrahmbhatt, nice idea. I appreciate any help 🙇

**Comment by DarkCodeMaster** [comment]
If I want to make a contribution to this repository, like translated into Chinese, How can I do?

**Reply by aleshkanet** [reply]
Please check it out: https://github.com/inancgumus/learngo/tree/master/translation/chinese

**Reply by DarkCodeMaster** [reply]
> Please check it out: https://github.com/inancgumus/learngo/tree/master/translation/chinese

Thank you!

**Comment by Preetiraj3697** [comment]
Thankyou
